src/SpectraProcessor.js

```diff
@@ -8,7 +8,8 @@ import { getNormalizedChart } from './jsgraph/getNormalizedChart';
 import { getPostProcessedChart } from './jsgraph/getPostProcessedChart';
 import { getTrackAnnotation } from './jsgraph/getTrackAnnotation';
 import { getCategoriesStats } from './metadata/getCategoriesStats';
-import { getClasses } from './metadata/getClasses';
+import { getClassLabels } from './metadata/getClassLabels';
+import { getClassesAsNumber } from './metadata/getClassesAsNumber';
```

Comment:
If we don't want a breaking change the name should stay getClasses and can not be renamed to getClassesAsNumber

Reply:
Got it!